backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/imagegallery/BpkImageGallery.kt

```diff
@@ -51,9 +53,113 @@ fun BpkImageGallery(
     }
 }
 
+@Composable
+fun BpkImageGalleryChipGrid(
+    categories: List<BpkImageGalleryChipCategory>,
+    closeContentDescription: String,
+    onCloseClicked: () -> Unit,
+    onDismiss: (() -> Unit),
+    modifier: Modifier = Modifier,
+    initialCategory: Int = 0,
+    onCategoryChanged: ((BpkImageGalleryChipCategory) -> Unit)? = null,
+    onImageClicked: ((BpkImageGalleryChipCategory, BpkImageGalleryImage) -> Unit)? = null,
+    onImageChanged: ((BpkImageGalleryChipCategory, BpkImageGalleryImage) -> Unit)? = null,
+    state: BpkModalState = rememberBpkModalState(),
+) {
+    BpkImageGalleryGridModal(
+        categories = BpkImageGalleryCategories.Chip(categories),
```

Comment:
an internal type to use here - this avoids the double nesting for consumers while still keeping things type safe internal